_data/vendors.yml

```diff
@@ -199,6 +199,14 @@
   percent_increase: 100%
   pricing_source: https://newrelic.com/products/infrastructure/pricing
   updated_at: 2018-10-18
+  
+- name: Notion
+  url: https://www.notion.so
+  base_pricing: $8 per u/m
+  sso_pricing: $20 per u/m
```

Comment:
@tim-fitzgerald the $20 pricing isn't listed on this pricing page, so it needs a source – was this a quote?